src/main/java/cs2103/v15_1j/jimjim/command/ChangeCommand.java

```diff
@@ -129,6 +130,8 @@ public UIFeedback execute(ControllerStates conStates) {
             	}
             }
             
+            conStates.masterList.sort();
```

Comment:
How about removing the item and then adding it back again? Then we don't need to write `sort` method for DataList